docs/lesson-plans/week-1.md

```diff
@@ -0,0 +1,649 @@
+a PostgreSQL database driver, [Jackson](https://github.com/FasterXML/jackson)
+JSON utilities, [JUnit 5](https://junit.org/junit5/) for a testing framework, and
+[Mockito](https://site.mockito.org/) for a mocking framework.
+
+Throughout the course, we'll be covering a lot of topics like Git and terminal commands,
+HTML, CSS, and JavaScript, servers, HTTP and REST requests, testing, databases, APIs, and
+cloud deployment. 
+
+Don't worry if any of this seems unfamiliar or new to you, we'll be sure to cover all of this
+and answer any questions you may have.
+
+### How are the projects organized?
+
+When you open up the project root folders, you'll see a bunch of different files and folders,
+probably a lot with strange names and filetypes you've never seen before. We'll go over quick
+descriptions of the files right now. You'll learn a lot more about everything below in the 
+coming weeks, so don't worry if you don't know what we're saying.
+
+#### Common Files
+
+.gitignore is a file containing regular expressions to describe files we don't want Git
+to keep track of when we make commits or push to GitHub.
+
+>A regular expression (regex) is a syntax used to define text matching the given pattern. 
+>Check out [regexr](https://regexr.com/) and [regex101](https://regex101.com/) for some 
+>common regexes, descriptions, and a playground to test some out.
+
+LICENSE is a file detailing licensing information. In our case, we're using the GPL-3.0
+License, which means anyone can use or modify our code for whatever they want (with some
+conditions). You'll rarely have to deal with this in practice, but we wanted to add one
+so that you can see what it looks like.
+
+README.md is sort of like the introduction file. It's written in a format called 
+[Markdown](https://en.wikipedia.org/wiki/Markdown) (the .md filetype), which allows for 
+simple and easy formatting. Usually this file has descriptions of the software, instructions
+for setting it up, and any other information the developer(s) would like to provide to the
+end user.
+
+#### Frontend
+
+node_modules/ is a directory which houses all of the JavaScript packages you'll need. You should
+never have do anything with that directory (other than deleting it if you're getting weird issues,
+but that should be __*RARE*__). This will appear after you've run `npm install` later on. This
+directory is also usually not included in a Git repository.
+
+public/ is a directory to store unique files and images like manifest files, HTML files, or
+special image files. They're put here for easy reference and access.
+
+src/ is the directory with your actual project files. All of your React components,
+views, tests, and code will be inside here.
+
+package.json is a file containing dependency/package information, scripts to be run using npm,
+and other data about your project.
+
+package-lock.json holds information on dependencies which dependencies require and special 
```

Comment:
adding commas will make this easier to read:
package-lock.json holds information on dependencies, which dependencies [are] require[d], and special...

Reply:
Whoops, that was a poorly constructed sentence. I went through and adjusted the wording a bit.